src/types/hermesProfileInterfaces.ts

```diff
@@ -0,0 +1,42 @@
+import { SharedEventProperties } from './chromeEventInterfaces';
+
+/**
+ * Each item in the stackFrames object of the hermes profile
+ */
+export interface HermesStackFrame {
+  line: string;
+  column: string;
+  funcLine: string;
+  funcColumn: string;
+  name: string;
+  category: string;
+  /**
+   * A parent function may or may not exist
+   */
+  parent?: number;
+}
+/**
+ * Each item in the samples array of the hermes profile
+ */
+export interface HermesSample {
```

Comment:
Define the interface for each Hermes Sample as derived from the sample hermes profile